content/reviewers_response.md

```diff
@@ -0,0 +1,642 @@
+
+**We report each token with a 95% confidence interval (CI). 
+Our frequency counts are sufficiently large that all calculated p-values are approximately or exactly zero.**
+
+> I would welcome a supplemental analysis, that removes single letters and special characters from the analysis of Figure 1B-E as they might change the baseline.
+
+**We have added this analysis to our supplement.**
+
+> The word cloud of Figure 2B, C is somewhat nice as it shows the main words. However, this information could also be conveyed in the text. Would personally favor to quantitatively see loadings of first few principal components for different terms.
+
+**We have provided a table in our supplemental section.**
+
+> The definition of "True matches" could be more explicit in within the main text as the preceding figure 3A could for some people set up a different anticipation.
+
+**We agree this term is too vague, and we have updated the text to be more explicit.**
+
+```diff
+- Approximately 98% of our 200 pairs with an embedding distance in the 0-25th and 25th-50th percentile bins were scored as true matches (Figure 3B).
+
++ Approximately 98% of our 200 pairs with an embedding distance in the 0-25th and 25th-50th percentile bins were successfully matched with their published counterpart (Figure 2B)
+```
+
+> The association given in Figure 4A seems to mainly stem from a few papers with large distances. Would an association be present when using the rank-based Spearman correlation instead of a linear regression? Would, for visualization, a logarithmic relationship describe the data better than a linear one?
+
+**We believe there is a misconception about our figure. 
+This figure depicts the median half-life publication time for each preprint category within bioRxiv.**
+
+> I believe that the analysis of Figure 4 B is quite clever as it would seem to address the thinkable concern of preprints with no delay and changes mainly stemming from those manuscripts that were already essentially accepted by manuscripts at the time of posting.
+
+**We are glad you found our analysis insightful.**
+
+> The analysis remarks that for the "Preprints in Motion Collection" the relationship between textual distance and time to publication disappears, and supports this through Figure 6E. However, the background trend in figure 6E includes publications that have been published at a time that exceeds a year. Hence a more faithful comparison would be to censor the background data by a distribution of durations that would correspond to the distribution of durations that would be possible for the "Preprint in Motions Collection" (taking distribution corresponding to interval between their dates on bioRxiv and the time at which authors assessed whether manuscripts were published).
+
+**We have provided a supplemental analysis within our manuscript that shows results after filtering the background distribution based on publication time.**
```

Comment:
```suggestion
**This point is very interesting!
The preprints in motion selection included preprints that were both posted within a specific time interval and then subsequently published.
We think the best comparison set would be with preprints posted on the same dates as preprint in motion preprints, but where the analysis is conducted on the set after enough years have passed for those that will be published to have been published.
Since we don't have this set, we did conduct the analysis proposed by the reviewer.
The results of this analysis are available in Supplemental Figure XYZ.**
```
Update to include the figure number to save the reviewer search time :)